ckan-2.10/base/Dockerfile.py3.10

```diff
@@ -69,19 +69,31 @@ RUN pip3 install -U pip && \
     ckan config-tool ${CKAN_INI} "beaker.session.secret = " && \
     ckan config-tool ${CKAN_INI} "ckan.plugins = ${CKAN__PLUGINS}"
 
-# Create a local user and group plus set up the storage path
-RUN groupadd -g 92 ckan && \
-    useradd -rm -d /srv/app -s /bin/bash -g ckan -u 92 ckan && \
-    mkdir -p ${CKAN_STORAGE_PATH} && \
-    chown -R ckan:ckan ${CKAN_STORAGE_PATH}
-
+# Create ckan and ckan-sys users and the ckan-sys group plus set up the storage path
+RUN groupadd -g 502 ckan-sys && \
+    useradd -rm -d /srv/app -s /bin/bash -g ckan-sys -u 502 ckan-sys && \
+    useradd -rm -d /srv/app -s /bin/bash -g ckan-sys -u 503 ckan
```

Comment:
having home directories owned by the user is nice for keeping track of things like `.bash_history` when logging in to the container

Reply:
if the ckan user has the same primary group as ckan-sys we'll need to make sure the group permissions on all the ckan source files prevent writing

Reply:
Yes, it could complicate things...actually it **would** complicate things. I'll drill down a bit more and think about this a bit more